packages/analytics-js/src/components/configManager/ConfigManager.ts

```diff
@@ -69,6 +85,8 @@ class ConfigManager implements IConfigManager {
       if (state.loadOptions.value.configUrl) {
         state.lifecycle.sourceConfigUrl.value = `${state.loadOptions.value.configUrl}/sourceConfig/?p=${MODULE_TYPE}&v=${APP_VERSION}&writeKey=${state.lifecycle.writeKey.value}&lockIntegrationsVersion=${lockIntegrationsVersion}`;
       }
+      // Set consent manager plugin name in state
+      state.consents.consentManager.value = consentManagerPluginName;
```

Comment:
We might need to store the consent manager options here.

Reply:
Consent manager option we already have in load options right?

Reply:
Then the definition of this state variable is ambiguous

Reply:
activeConsentProvider sound good?

Reply:
Yeah, that seems appropriate.